rivet.spec

```diff
@@ -16,6 +17,7 @@ Patch0: rivet-1.4.0
 
 %prep
 %setup -n rivet/%{realversion}
+#%setup -n Rivet-%{realversion}
 %patch0 -p0
 
 ./configure --disable-silent-rules --prefix=%i --with-boost=${BOOST_ROOT} --with-hepmc=$HEPMC_ROOT \
```

Comment:
You just removed `boost` and `yaml-cpp`from run-time dependencies, but you ask to build with it at configure time. Clean it up.